src/app/home/directives/home-story.component.ts

```diff
@@ -95,4 +65,30 @@ export class HomeStoryComponent implements OnInit {
     }
   }
 
+  get isPlusSign(): boolean {
+    return this.story.isNew && !this.story.isStored()
```

Comment:
i was purposefully avoiding calling `changed()` earlier, as i didn't want to load-related.  But now that we have to... may as well.